### PR TITLE
Map PR new map implementation

### PR DESCRIPTION
**Pull request by PeterXRiber**
New map implementation

### CODE REVIEW

**Comment by OFHansen** [comment]
Remove unnecessary comments and change the needed ones into English

**Comment by PeterXRiber** [comment]
> Remove unnecessary comments and change the needed ones into English

Done

File: common/src/main/java/abstractClasses/Entity.java [hunk 1]
```diff
@@ -68,8 +75,20 @@ public void setSprite(URL sprite, double[] scale) {
...
     }
 
+    public void setSprite(BufferedImage bufferedImage, double[] scale) {
+        this.scale = scale;
+        this.sprite = ImageLoader.loadImage(bufferedImage, this.scale);
+    }
+
     public void setPosition(int[] position) {
         this.position = position;
+        // tjekker om der kun er 1 ting i arrayet
```
Comment (OFHansen) [inline common/src/main/java/abstractClasses/Entity.java:85]: missed this

File: common/src/main/java/interfaces/IGameEngine.java [hunk 2]
```diff
@@ -11,6 +11,7 @@ public interface IGameEngine {
     LinkedList<IProcessing> getProcesses();
     boolean addDrawables(IDrawable drawable);
     boolean addDrawables(IDrawable drawable, Layers layer);
+    // hvis ikke layer gives - vil addDrawables sætte indputtet (aka et map) til "MIDDLEGROUND"
```
Comment (OFHansen) [inline common/src/main/java/interfaces/IGameEngine.java:14]: missed this

**Review by OFHansen — CHANGES_REQUESTED** [review]

**Comment by OFHansen** [comment]
Nice

**Pull request merged**